setup.py

```diff
@@ -40,6 +40,7 @@
             "pytest",
             "pyyaml",
             "torchtestcase",
+            "scikit-learn",
```

Comment:
This should also go into `environment.yml` (which should split out test dependencies, see #181.

Reply:
Is already in `environment.yml`. But I agree, we should split test dependencies there as well. 
https://github.com/mackelab/sbi/blob/1d32e5232790b05a82b4fbad67f7138eb4691198/environment.yml#L40